nntrainer/tensor/blas_neon.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include <blas_neon.h>
 #include <nntrainer_error.h>
+#include <omp_setting.h>
```

Comment:
I guess openMP Support is a prerequisite of  NNTrainer Then. It is ok because most of compilers support OpenMP.